data/mods/Aftershock/maps/mapgen_pallete/houses.json

```diff
@@ -0,0 +1,96 @@
+[
+  {
+    "type": "palette",
+    "//": "the food should be rotted, but for now the regular food item groups can work for now",
+    "id": "standard_afsshelter_palette",
+    "toilets": { "t": {  } },
+    "terrain": {
+      " ": "t_region_groundcover_urban",
+      ".": "t_metal_floor",
+      "+": "t_wall_prefab_metal",
+      "o": [
+        [ "t_wall_prefab_glass", 2 ],
+        [ "t_wall_prefab_glass_shutters", 2 ],
+        [ "t_wall_prefab_bglass_shutters", 4 ],
+        [ "t_wall_prefab_bglass", 5 ]
+      ],
+      "g": "t_wall_metal",
+      "h": "t_concrete_wall",
+      "i": [ [ "t_door_glass_c", 5 ], "t_door_glass_o", 5 ],
+      "j": [ [ "t_door_locked", 1 ], [ "t_door_c", 5 ], [ "t_door_o", 1 ] ],
+      "x": [ [ "t_door_locked_interior", 1 ], [ "t_door_c", 5 ], [ "t_door_o", 5 ] ],
+      "l": [ [ "t_door_metal_c", 5 ], [ "t_door_metal_pickable", 3 ], [ "t_door_metal_locked", 1 ], [ "t_door_metal_o", 5 ] ],
+      "n": "t_metal_ventilation_shutter",
+      "p": "t_metal_railing",
+      "<": "t_stairs_down",
+      ">": "t_stairs_up"
+    },
+    "furniture": {
+      "@": "f_bed",
+      "D": "f_wardrobe",
+      "9": "f_shower",
+      "5": "f_sink",
+      "f": "f_table",
+      "y": "f_table",
+      "h": "f_chair",
+      "P": "f_dishwasher",
+      "i": "f_bench",
+      "2": "f_coffee_table",
+      "F": "f_fridge",
+      "H": "f_sofa",
+      "L": "f_cupboard",
+      "{": "f_cupboard",
+      "V": "f_cupboard",
+      "I": "f_cupboard",
+      "^": "f_sink",
+      "M": "f_cupboard",
+      "S": "f_cupboard",
+      "J": "f_counter",
+      "q": "f_rack",
+      "O": "f_oven",
+      "R": "f_bookcase",
+      "Y": "f_trashcan",
+      "E": "f_armchair",
+      "7": "f_locker",
+      "W": "f_washer",
+      "Z": "f_dryer"
+    },
+    "items": {
+      "t": { "item": "SUS_toilet", "chance": 10, "repeat": [ 1, 3 ] },
+      "D": [ { "item": "wardrobe_unisex_afs", "chance": 100, "repeat": [ 1, 2 ] } ],
+      "O": { "item": "SUS_oven", "chance": 100 },
+      "2": [
+        { "item": "consumer_electronics", "chance": 10, "repeat": [ 1, 3 ] },
+        { "item": "livingroom", "chance": 10, "repeat": [ 1, 3 ] }
+      ],
+      "F": { "item": "SUS_fridge", "chance": 80 },
```

Comment:
Perhaps a freezer should be built into an outside wall, to take advantage of external temperatures?

Reply:
Maybe in the future? there is no dedicated freezer furniture afaik right now so it'll have to be added